c2_components/src/mfx_c2_encoder_component.cpp

```diff
@@ -1133,7 +1161,9 @@ void MfxC2EncoderComponent::WaitWork(std::unique_ptr<C2Work>&& work,
             std::unique_ptr<C2Worklet>& worklet = work->worklets.front();
 
             worklet->output.flags = work->input.flags;
-            if (!m_bHeaderSent) {
+
+            // VPX unsupport SPS PPS
+            if (!m_bHeaderSent && m_encoderType != ENCODER_VP9) {
```

Comment:
Please change it like this, currently, only h264 or h265 needs the header info. 
if (!m_bHeaderSent && (m_encoderType == ENCODER_H264 || m_encoderType == ENCODER_H265))